cluster/providers/install-provider.sh

```diff
@@ -56,6 +58,13 @@ case $PROVIDER_NAME in
     #create sample VMs and volume disks for the tests
     ${SCRIPT_DIR}/openstack/create_test_vms.sh
     ;;
+  "ova")
+    echo "installing ova providers"
+    # installs NFS for CSI
+    [ ! -z "${INSTALL_NFS}" ] && ${SCRIPT_DIR}/openstack/install_nfs.sh
```

Comment:
openstack?

Reply:
yep, its the same script used bu all providers, its simply located under the openstack folder.

Reply:
then maybe it worth pulling it out and everyone will use share place? it is a bit confusing.